### DISCUSSION TITLE
How to map user attributes by group in Access Token/ID Token

### DISCUSSION THREAD

**Discussion by hantsy** [keycloak/keycloak#30848] [Q&A]
I created an `app_metadata` group for the user attributes that are related to our business logic. 

What I want to map in access token like this.

```json
{
  // other data in token
  "app_metadata": {
    "key1": "value1", 
    "key2": "value2"
  }
}
```
But there is no mapper to map the attributes in this group to access token directly, I have to map then one by one like this.

```json
{

  "key1": "value1", 
  "key2": "value2"
}
```

**Comment by syedmehdifaraz** [comment]
@hantsy 
I am facing a similar issue and wanted to check if you were able to resolve the issue.
